samples/nrf9160/mqtt_simple/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@
 
 cmake_minimum_required(VERSION 3.8.2)
 
+include(../../../cmake/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
@carlescufi @SebastianBoe This line is added to pick up boards defined only in the -nrf repo, in particular PCA20035. Not sure if this is the right way to go, as it would have to be added to every sample. Could you please advise how to do this properly?

Reply:
This is the current best-practice.

Now that west is released we could eventually check for additional BOARD_ROOT's from app/boilerplate.cmake, but for now this is best-practice.

Reply:
Alright, then I'll keep it as is. Thanks!